bootstrap/bootstrap.py

```diff
@@ -79,10 +89,10 @@ def __init__(self, ghc_path: Path):
         self.version = info['Project version']
         #self.lib_dir = Path(info['LibDir'])
         #self.ghc_pkg_path = (self.lib_dir / 'bin' / 'ghc-pkg').resolve()
-        self.ghc_pkg_path = (self.ghc_path.parent / 'ghc-pkg').resolve()
+        self.ghc_pkg_path = (self.ghc_path.parent / 'ghc-pkg.exe').resolve()
```

Comment:
The `.exe` suffix should probably be platform dependent?

Reply:
oops I committed that by mistake, what's the usual way to do this in Python?

Reply:
```python
        exe = ''
        if platform.system() == 'Windows': exe = '.exe'

        self.ghc_pkg_path = (self.ghc_path.parent / 'ghc-pkg' + exe).resolve()
```

?

Reply:
That doesn't work, nor does `os.path.join(.., exe)`

Reply:
OK done

Reply:
yeah, that seems fine.

Do we need the same change in the GHC bootstrap script to get it working on windows?

Reply:
yes, that's in GHC MR !8821